build/config/compiler/BUILD.gn

```diff
@@ -346,7 +346,7 @@ config("rtti_default") {
 }
 
 config("no_exceptions") {
-  cflags = [ "-fno-exceptions" ]
```

Comment:
Why was it needed? Whatever it was, we need another workaround. :)

Reply:
This is only going to be temporary, because we still need to make the building process more dynamic. The reason why I removed this flag from compilation is because pybind11 uses exceptions in C++.